src/api/Shrooms.Premium/Domain/Services/Events/List/EventListingService.cs

```diff
@@ -188,7 +210,12 @@ public async Task<IPagedList<EventParticipantReportDto>> GetReportParticipantsAs
                             StartDate = visited.StartDate,
                             EndDate = visited.EndDate
                         })
-                        .ToList()
+                        .ToList(),
+                    TotalVisitedEventCount = _eventsDbSet
+                        .Count(e => e.EventParticipants.Any(participant => participant.ApplicationUserId == p.ApplicationUser.Id) &&
+                                    e.EndDate < DateTime.UtcNow &&
+                                    e.OrganizationId == userOrg.OrganizationId &&
+                                    (eventTypesLength == 0 || reportArgsDto.EventTypeIds.Contains(e.EventType.Id)))
```

Comment:
Tried to export this LINQ to Expression<Func<Event, bool>> (in this method we have the same expression twice) function, however I kept getting this error.
![Screenshot 2022-07-14 104859](https://user-images.githubusercontent.com/75424325/179162169-32362e39-c778-4123-8d03-1e92a8ebaa09.png)

Reply:
Which EF version does Simoona use? it seems there is a bug with making queries like that for navigation properties in EF 6, and also what happens if you only move the bool check inside the Any() to a separate method?

Reply:
Seems like the version is 6.1.3. 

Exporting Any() I get:
![image](https://user-images.githubusercontent.com/75424325/179766904-dd36e515-3a10-4f92-be29-6bf1168b3be4.png)